build-support/compile_time.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
this is a new file, should use starrocks' license header.

Reply:
Updated